modules/microshift-kubeconfig-remote-con.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_networking/microshift-kubeconfig.adoc
+
+:_content-type: CONCEPT
+[id="remote-access-con_{context}"]
+= Remote access kubeconfig files
+
+Remote access `kubeconfig` file values can be configured. {product-title} generates a default `kubeconfig` for external access using the `hostname` value. An additional `kubeconfig` file is generated for each entry in the `apiServer.subjectAltNames` fields. You can copy these files from the host during times of IP connectivity and then implement them from other workstations.
```

Comment:
I was going to leave a comment here saying that we need to talk about "why" because without it this "what" information doesn't tell the user very much. The "why" comes later, so maybe this paragraph just needs to move down around what is now line 33?